features/font-variant-alternates.yml

```
@@ -3,3 +3,33 @@ description: The `font-variant-alternates` CSS property, along with the `@font-f
 spec: https://drafts.csswg.org/css-fonts-4/#font-variant-alternates-prop
 group: fonts
 caniuse: font-variant-alternates
+status:
+  compute_from: css.at-rules.font-feature-values
```

This aligns better with caniuse, which is nice! That said, I think it's slightly surprising that we're not pinning to the thing named in the feature, which shares the same compatibility story. This might be a good place to use two items:

```suggestion
  compute_from:
    - css.properties.font-variant-alternates
    - css.at-rules.font-feature-values
```